packages/powersync_flutter_libs/lib/powersync_flutter_libs.dart

```diff
@@ -0,0 +1,131 @@
+
+import 'dart:async';
+import 'dart:ffi';
+import 'dart:io';
+import 'dart:isolate';
+
+import 'powersync_flutter_libs_bindings_generated.dart';
+
+/// A very short-lived native function.
+///
+/// For very short-lived functions, it is fine to call them on the main isolate.
+/// They will block the Dart execution while running the native function, so
+/// only do this for native functions which are guaranteed to be short-lived.
+int sum(int a, int b) => _bindings.sum(a, b);
+
+/// A longer lived native function, which occupies the thread calling it.
+///
+/// Do not call these kind of native functions in the main isolate. They will
+/// block Dart execution. This will cause dropped frames in Flutter applications.
+/// Instead, call these native functions on a separate isolate.
+///
+/// Modify this to suit your own use case. Example use cases:
+///
+/// 1. Reuse a single isolate for various different kinds of requests.
+/// 2. Use multiple helper isolates for parallel execution.
+Future<int> sumAsync(int a, int b) async {
+  final SendPort helperIsolateSendPort = await _helperIsolateSendPort;
+  final int requestId = _nextSumRequestId++;
+  final _SumRequest request = _SumRequest(requestId, a, b);
+  final Completer<int> completer = Completer<int>();
+  _sumRequests[requestId] = completer;
+  helperIsolateSendPort.send(request);
+  return completer.future;
+}
+
+const String _libName = 'powersync_flutter_libs';
+
+/// The dynamic library in which the symbols for [PowersyncFlutterLibsBindings] can be found.
+final DynamicLibrary _dylib = () {
```

Comment:
How does this related to `Abi.current().localName`? It looks like two variations on almost the same thing.

Reply:
This was generated by the flutter template. It is now removed.